virttest/utils_conn.py

```diff
@@ -1939,45 +2005,59 @@ def conn_setup(self):
                                }
             self.client_hosts.sub_else_add(pattern_to_repl)
             pattern_to_repl = {r".*sasl_allowed_username_list\s*=.*":
-                               'sasl_allowed_username_list=%s' % sasl_allowed_username_list}
+                               'sasl_allowed_username_list=%s'
+                               % sasl_allowed_username_list}
             self.client_libvirtdconf.sub_else_add(pattern_to_repl)
-        else:
+        elif self.sasl_type != 'plain':
```

Comment:
Do we need to cover 'else' here as in the previous code?

Reply:
Else statement from here is covered on line 2017.